blocks/init/src/Blocks/components/drawer/assets/drawer.js

```diff
@@ -26,38 +22,33 @@ export class Drawer {
 	}
 	
 	openMobileMenu() {
-		if (this.overlay) {
-			document.body.classList.add(this.CLASS_OVERLAY_IS_SHOWING);
-			this.trigger?.classList.add(this.CLASS_MENU_OPEN);
-		}
+		this.trigger?.classList.add(this.CLASS_MENU_OPEN);
+		this.drawerElement.style.display = 'block';
+
+		const addClass = debounce(() => document.body.classList.add(this.CLASS_MENU_OPEN), 300);
```

Comment:
Why is this debounced?

Reply:
I believe it's here to prevent another "opening" of the menu while the animation is ongoing. However, I would suggest doing a monkey lock in combination with the 'transitionend' listener.
That way you avoid open/close collision and you avoid setting the exact time for animation/transition duration.